spot_driver/spot_driver/spot_ros2.py

```diff
@@ -2919,6 +2929,43 @@ def handle_set_gripper_camera_parameters(
 
         return response
 
+    def handle_arm_velocity_command(self, arm_velocity_command: ArmVelocityCommandRequest) -> None:
+        if self.spot_wrapper is None:
+            return
+
+        cylindrical_velocity = arm_command_pb2.ArmVelocityCommand.CylindricalVelocity()
+        cylindrical_velocity.linear_velocity.r = arm_velocity_command.command.cylindrical_velocity.linear_velocity.r
+        cylindrical_velocity.linear_velocity.theta = (
+            arm_velocity_command.command.cylindrical_velocity.linear_velocity.theta
+        )
+        cylindrical_velocity.linear_velocity.z = arm_velocity_command.command.cylindrical_velocity.linear_velocity.z
+
+        angular_velocity = geometry_pb2.Vec3(
+            x=arm_velocity_command.angular_velocity_of_hand_rt_odom_in_hand.x,
+            y=arm_velocity_command.angular_velocity_of_hand_rt_odom_in_hand.y,
+            z=arm_velocity_command.angular_velocity_of_hand_rt_odom_in_hand.z,
+        )
+
+        cartesian_velocity = arm_command_pb2.ArmVelocityCommand.CartesianVelocity()
+        cartesian_velocity.velocity_in_frame_name.x = (
+            arm_velocity_command.command.cartesian_velocity.velocity_in_frame_name.x
+        )
+        cartesian_velocity.velocity_in_frame_name.y = (
+            arm_velocity_command.command.cartesian_velocity.velocity_in_frame_name.y
+        )
+        cartesian_velocity.velocity_in_frame_name.z = (
+            arm_velocity_command.command.cartesian_velocity.velocity_in_frame_name.z
+        )
+        cartesian_velocity.frame_name = arm_velocity_command.command.cartesian_velocity.frame_name
+
+        proto_command = arm_command_pb2.ArmVelocityCommand.Request(
+            cylindrical_velocity=cylindrical_velocity,
+            angular_velocity_of_hand_rt_odom_in_hand=angular_velocity,
+            cartesian_velocity=cartesian_velocity,
+        )
```

Comment:
You should be able to replace this with the following, using some built in proto to ros conversion:
```
proto_command = arm_command_pb2.ArmVelocityCommand.Request()
convert(arm_velocity_command, proto_request)
```
Can you verify that this still works with your example?

Reply:
Will test it later this afternoon on the robot :)

Reply:
Hello, just wondering if you had a chance to test this out?